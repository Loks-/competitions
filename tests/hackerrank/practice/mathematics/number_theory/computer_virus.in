5
2 0
2 1
2 2
3 1
1 10
