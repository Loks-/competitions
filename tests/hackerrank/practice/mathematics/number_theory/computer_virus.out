1
5
13
7
21
